sftp/1.3/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,40 @@
+= SFTP Connector 1.3 - Mule 4
+:page-aliases: connectors::sftp/sftp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for SFTP (SFTP Connector) provides access to files and folders on a SFTP server. SFTP connector operations manage file transfers over the SFTP (Secure File Transfer Protocol) protocol. The connector main features include:
+
+* Read files or list directory contents on demand.
+* Support for common SFTP operations such as creating directories, copying, moving, renaming, deleting files.
+* Support for locking files.
+* File matching functionality.
+* Consistent design with the File and FTP Connectors. The same set of operations is available on these connectors, and they behave similarly.
+
+=== Representation of File Attributes
+
+Mule 4 provides a message structure composed of a payload and attributes, where the data moves around. This means that the payload is a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, timestamps, and so on. For the FTP and SFTP connectors these attributes are known as the _file attributes_.
```

Comment:
```suggestion
Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
```
GLOBALLY omit ", and so on." It is vague and also unnecessary if the text includes "such as," which implies some but not all.